.travis.yml

```diff
@@ -122,12 +134,13 @@ matrix:
         # we can spot issues sooner. We do not use remote data here, since
         # that gives too many false positives due to URL timeouts.
         - os: linux
+          stage: cron_only
           env: NUMPY_VERSION=dev
                CONDA_DEPENDENCIES=$CONDA_ALL_DEPENDENCIES
-               EVENT_TYPE='cron'
 
     allow_failures:
-      - env: NUMPY_VERSION=dev SETUP_CMD='test --remote-data'
+      - os: linux
+        env: NUMPY_VERSION=dev SETUP_CMD='test --remote-data'
```

Comment:
Should this include ``stage: cron_only``? More generally this doesn't match exactly the entry above, so I think that could be why it's not recognized (e.g. SETUP_CMD is different here)

Reply:
There are two dev tests, same as before. I think this is the same as the first one, or at least the intent was that they should be the same.

We always run one on each commit, but allow it to fail. However from cron, we don't allow it to fail, so the maintainers will get the notification from travis (which at the moment is not very useful as the build is always failing due to numpy is not ready to decide about the style issue).